server/pulp/server/db/manage.py

```diff
@@ -191,6 +202,29 @@ def main():
         options = parse_args()
         _start_logging()
         connection.initialize(max_timeout=1)
+
+        if status.get_workers():
+            last_worker_timestamp = max([worker['last_heartbeat'] for worker
+                                         in status.get_workers()])
+            wait_time = timedelta(seconds=constants.CELERY_TIMEOUT_SECONDS) \
+                - (UTCDateTimeField().to_python(datetime.now()) - last_worker_timestamp)
+
+            if wait_time > timedelta(0):
+                print >> sys.stdout, _('\rThe following processes might still be running %s'
```

Comment:
Can these print calls be converted to not use the `>>` operator. I can't find the documentation for that in the Python docs even. Changing it to `print 'stuff here'` would be consistent with the rest of the codebase.